### ISSUE TITLE
ConvertTo-Bicep throws error when using -AsString switch

### ISSUE THREAD

**Issue by StefanIvemo** [StefanIvemo/BicepPowerShell#165]
**Describe the bug**
`ConvertTo-Bicep -AsString` throws an error.  This is because even though we output the bicep data as a string the following is executed `$null = Build-Bicep -Path $FilePath -AsString`. We need to exclude it when `-AsString` is used. 

**To Reproduce**
```powershell
convertto-bicep C:\git\StefanIvemo\BicepPowerShell\Tests\supportFiles\workingARMTemplate.json -AsString
WARNING: Decompilation is a best-effort process, as there is no guaranteed mapping from ARM JSON to Bicep.
You may need to fix warnings and errors in the generated bicep file(s), or decompilation may fail entirely if an accurate conversion is not possible.        
If you would like to report any issues or inaccurate conversions, please see https://github.com/Azure/bicep/issues.
param name string
param location string = resourceGroup().location

resource name_resource 'Microsoft.Storage/storageAccounts@2019-06-01' = {
  name: name
  tags: {
    displayName: 'storageaccount1'
  }
  location: location
  kind: 'StorageV2'
  sku: {
    name: 'Premium_LRS'
    tier: 'Premium'
  }
}
Get-ChildItem:
Line |
  71 |          $files = Get-Childitem -Path $Path *.bicep -File
     |                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | Cannot find path 'C:\git\StefanIvemo\BicepPowerShell\Tests\supportFiles\workingARMTemplate.bicep' because it does not exist.
```

**Expected behavior**

**Additional context**

**Comment by emilguden** [comment]
Can't reproduce this issue - can you please confirm?

This is what my environment looks like.

```powershell
ConvertTo-Bicep C:\temp\asp.json -AsString
WARNING: Decompilation is a best-effort process, as there is no guaranteed mapping from ARM JSON to Bicep.
You may need to fix warnings and errors in the generated bicep file(s), or decompilation may fail entirely if an accurate conversion is not possible.
If you would like to report any issues or inaccurate conversions, please see https://github.com/Azure/bicep/issues.
param name string
param skuName string
param tier string
param size string
param family string

resource name_resource 'Microsoft.Web/serverfarms@2021-01-01' = {
  name: name
  location: resourceGroup().location
  sku: {
    name: skuName
    tier: tier
    size: size
    family: family
  }
}

output aspID string = name_resource.id
C:\temp\asp.bicep(7,24) : Warning BCP081: Resource type "Microsoft.Web/serverfarms@2021-01-01" does not have types available.
```

```powershell
$PSVersionTable

Name                           Value
----                           -----
PSVersion                      7.1.3
PSEdition                      Core
GitCommitId                    7.1.3
OS                             Microsoft Windows 10.0.19042
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

```powershell
Get-Module Bicep

ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     1.4.7                 Bicep                               {Build-Bicep, Convert-Bice…
```

**Comment by StefanIvemo** [comment]
Fixed in #188
